lib/datadog/opentracer/tracer.rb

```diff
@@ -15,10 +15,14 @@ class Tracer < ::OpenTracing::Tracer
       attr_reader \
         :datadog_tracer
 
-      # (see Datadog::Tracer#initialize)
-      def initialize(**options)
+      # Create an OpenTracing::Trace instance that is bound to the active
+      # Datadog tracer instance.
+      # To configure this tracer, use the global tracing configuration.
+      #
+      # DEV-2.0: Remove options as configuration is done through the global tracing configuration.
+      def initialize(**_legacy_options)
         super()
-        @datadog_tracer = Datadog::Tracing::Tracer.new(**options)
```

Comment:
Minor: I wonder if it's worth adding a warning when `_legacy_options` is not empty, to warn people they're being ignored and they should stop passing them in.

Reply:
Done!